README.md

```diff
@@ -2,6 +2,15 @@
 
 Repository for out-of-tree scheduler plugins based on the [scheduler framework](https://kubernetes.io/docs/concepts/scheduling-eviction/scheduling-framework/).
 
+This repository includes some incubating ideas of solveing real problems which is hard to be solved with only in-tree scheduler plugins. 
+You can use it directly via our providing images, use it as the library to build your custom scheduler, or use it as an incubator to build new plugins for your ideas.
+
+Whether a plugin is an in-tree plugin or a plugin here should be determined by the following points:
```

Comment:
I think this part should not appear in the README here.
Whether a plugin is an in-tree, it is needed to decide on a case-by-case basis. Even though the following conditions are met, it does not determine whether it can be placed in the in-tree.